src/tribler/core/components.py

```diff
@@ -158,8 +159,8 @@ def finalize(self, ipv8: IPv8, session: Session, community: Community) -> None:
         """
         When we are done launching, register our REST API.
         """
-        session.rest_manager.get_endpoint("/api/downloads").mds = session.mds
-        session.rest_manager.get_endpoint("/api/statistics").mds = session.mds
+        session.rest_manager.get_endpoint("/api/downloads").mds = session
```

Comment:
This doesn't look right. Shouldn't this be `session.mds` still?

Reply:
You're right. I don't think `DownloadsEndpoint.mds` is actually used. anymore. That explains the lack of errors.